### ISSUE TITLE
Development Ministry shouldn't close after one entry is Canceled/Subbed

### ISSUE THREAD

**Issue by Imzogelmo** [plainblack/Lacuna-Server-Open#372]
Because, y'know, you may want to do other stuff while you have that window open.

**Comment by Wolfsfriend** [comment]
I agree, I have had a ship arrive with Halls on board and then decided to stop all upgrades to regular buildings so that I could upgrade my Glyph Buildings. As it is now I have to reopen the DM for each upgrade I wish to cancel.
